deploy/kubernetes/overlays/prow-pull/kustomization.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+bases:
+- ../../base
+images:
+- name: REPLACEME/gcp-compute-persistent-disk-csi-driver
+  newName: gcr.io/gke-release/gcp-compute-persistent-disk-csi-driver
+  newTag: "v0.4.0-gke.0"
+- name: REPLACEME/csi-provisioner
+  newName: quay.io/k8scsi/csi-provisioner
```

Comment:
the pull job should be using the gke release versions of sidecars right? and the `REPLACEME/gcp-compute-persistent-disk-csi-driver` should actually be replaced by a pull request specific built container?

Reply:
Ideally the pull job should use the gke versions of sidecars, however, we have a chicken and egg problem when we want to test a new version of a sidecar but it's not available in gke-release yet.  For example, [enabling topology](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/243).

The other option, is to not test new sidecar features in the pull job (only test stable features), and only switch over pull jobs when we change stable.  Actually, the pull job could just use the stable overlay.